src/Task2/Machine.java

```diff
@@ -0,0 +1,26 @@
+package Task2;
```

Comment:
the package's name should not be capitalized

Reply:
Okay